radio/src/targets/flysky/CMakeLists.txt

```diff
@@ -90,9 +101,15 @@ set(TARGET_SRC
   i2c_common_v2.cpp
   i2c_driver.cpp
   crc_driver.cpp
-  tools/elrsV2.cpp
+#  tools/elrsV2.cpp
```

Comment:
Like in other places, please leave it like it is. It is removed during compilation anyway.